src/reducers/index.js

```diff
@@ -12,9 +12,13 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+      return  {
+        count: initialState.count += action.payload 
```

Comment:
`initialState.count += action.payload` is an assignment statement, meaning you're mutating the initialState which is an anti pattern in React and Redux. What you can do here is to just use `state` object to access the current value of count, like:
```
  count: state.count + action.payload
```